OutOfSchool/OutOfSchool.IdentityServer/Controllers/AccountController.cs

```diff
@@ -421,11 +429,11 @@ public async Task<IActionResult> ChangePassword(ChangePasswordViewModel model)
 
     [HttpGet]
     [Authorize]
-    private async Task<IActionResult> SendConfirmEmailProcess(User user, string razorTemplate, object passedData)
+    private async Task<IActionResult> SendConfirmEmailProcess(string action, User user, string razorTemplate, object passedData)
```

Comment:
Please remove misleading attributes for this method. Framework doesn't route to private methods and thus never checks their attributes

Reply:
Done